Kernel/FileSystem/VirtualFileSystem.cpp

```diff
@@ -854,9 +870,12 @@ ErrorOr<void> VirtualFileSystem::rmdir(Credentials const& credentials, StringVie
         return {};
```

Comment:
Slightly unrelated, but I it should be possible to terminate the count once it reached 3.